harmony/__init__.py

```diff
@@ -7,7 +7,7 @@
 """
 
 # Automatically updated by `make build`
-__version__ = "0.0.1"
+__version__ = "1.0.8-alpha.6"
```

Comment:
I assume this was created by github? Are we ok with versions greater than 1?

Reply:
1.x.x is good, yes, and already published to PyPI.  This version was automatically created.  I'll update it to the latest published version before merge.